corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -200,7 +200,7 @@ <h4 class="modal-title">{% trans "Create a new Case Type" %}</h4>
       ></div>
     {% endif %}
     <div class="row">
-      <div class="col-md-12">
+      <div class="col-xs-12">
```

Comment:
Is there a world were you would want to apply both classes to the div so the xs one can take affect on smaller screens and the md can take effect on larger screens? Or is the idea that since they are the same width (12), we only need to specify the smallest style? I know I've seen some spots where we do multiple classes so just curious.